app/src/test/java/invoice_automation/module/QuickBooksModuleTest.java

```diff
@@ -1,15 +1,13 @@
 package invoice_automation.module;
 
 import com.intuit.ipp.core.Context;
-import com.intuit.ipp.data.Customer;
-import com.intuit.ipp.data.EmailAddress;
-import com.intuit.ipp.data.EmailStatusEnum;
-import com.intuit.ipp.data.Invoice;
-import com.intuit.ipp.data.MemoRef;
+import com.intuit.ipp.data.*;
```

Comment:
if you're using intellij can you follow the instructions on this page: 
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports
to stop wildcard imports? they pollute namespaces

Reply:
Made the change 👍